Create 2022-05-18-faststore.md (#1305)
vtex/faststore: docs/blog/2022-05-18-faststore.md

---
description: The FastStore API types, queries and mutations have been documented. You can check it now by using GraphiQL.
tags: [faststore]
hide_table_of_contents: false
---

# FastStore API documentation updated

> We have made available FastStore API reference documentation. You can check it now by using [GraphiQL](https://github.com/graphql/graphiql).

Reviewer: ```suggestion
FastStore API reference documentation is now available. You can check it via the [GraphiQL](https://github.com/graphql/graphiql) IDE.
```

## What has changed?

> When working on a FastStore project, you can run it locally and use **GraphiQL** to explore the API. Now, when you do this you will see that each type, argument, querie and mutation has an associated description.

Reviewer: ```suggestion
While working on your FastStore website, you can run your project locally and explore the API using **GraphiQL**. Now, when you do this, you will see that each type, argument, query, and mutation has an associated description.
```

## What needs to be done?

> To see these new docs follow these steps:

Reviewer: ```suggestion
To explore the FastStore API documentation, follow these steps:
```
